cmd/ctr/commands/content/fetch.go

```diff
@@ -177,7 +179,9 @@ func Fetch(ctx context.Context, client *containerd.Client, ref string, config *F
 	go func() {
 		if config.ProgressOutput != nil {
 			// no progress bar, because it hides some debug logs
-			ShowProgress(pctx, ongoing, client.ContentStore(), config.ProgressOutput)
+			if config.Verbose {
+				ShowProgress(pctx, ongoing, client.ContentStore(), config.ProgressOutput)
+			}
```

Comment:
Probably better to control `config.ProgressOutput` instead of adding a new `config.Verbose`.

Reply:
I've got it , but I will modify it according to your meaning first.